crates/augurs-forecaster/src/transforms.rs

```diff
@@ -7,15 +7,21 @@ use augurs_core::{
     Forecast,
 };
 
+/// Transforms and Transform implementations.
+///
+/// The `Transforms` struct is a collection of `Transform` instances that can be applied to a time series.
+/// The `Transform` enum represents a single transformation that can be applied to a time series.
 #[derive(Debug, Default)]
-pub(crate) struct Transforms(Vec<Transform>);
+pub struct Transforms(Vec<Transform>);
```

Comment:
I'm not sure this really needs to be `pub`, the `Forecaster` impl just accepts a `Vec<Transform>` and converts them to this to give us some convenience methods :thinking: